vspace/src/main/java/edu/asu/diging/vspace/core/model/IBranchingPoint.java

```diff
@@ -0,0 +1,12 @@
+package edu.asu.diging.vspace.core.model;
+
+import java.util.List;
+
+import edu.asu.diging.vspace.core.model.impl.Choice;
+
+public interface IBranchingPoint {
+
+	List<Choice> getChoices();
+	
+	void setChoices(List<Choice> choices);
```

Comment:
those methods need to return/work with the interfaces not the concrete implementations